be/src/exec/olap_table_info.h

```diff
@@ -105,12 +105,21 @@ struct OlapTablePartition {
 class OlapTablePartKeyComparator {
 public:
     OlapTablePartKeyComparator(SlotDescriptor* slot_desc) : _slot_desc(slot_desc) { }
+    // return true if lhs < rhs
+    // 'nullptr' is max value, but 'null' is min value
     bool operator()(const Tuple* lhs, const Tuple* rhs) const {
         if (lhs == nullptr) {
             return false;
         } else if (rhs == nullptr) {
             return true;
         }
+
+        if (lhs->is_null(_slot_desc->null_indicator_offset())) {
+            return true;
```

Comment:
if rhs->is_null() is false, this will return false

Reply:
Changed